index.bs

```diff
@@ -3530,6 +3530,26 @@ operation. Likewise, clients can choose to produce a [=client extension output=]
 understand by encoding the [=authenticator extension output=] value into JSON, provided that the CBOR output uses only types
 present in JSON.
 
+When clients choose to pass through extensions they do not recognize,
+the JavaScript values in the [=client extension input=] values
```

Comment:
(language nit: could go either way:) I think there are too many "values" here and would drop the instances that follow &ldquo;input=] &rdquo;. (Same below.)

Reply:
Thanks - I'll work on the exposition.